.eleventy.js

```diff
@@ -1,38 +1,53 @@
-const { isDev } = require('./pages/views/_data/env')
+const { isDev } = require("./pages/views/_data/env");
+const htmlmin = require("html-minifier");
 
-module.exports = config => {
+module.exports = (config) => {
   config.addPassthroughCopy({
-    'pages/static/assets': 'assets',
-    'pages/static/tools': '.',
+    "pages/static/assets": "assets",
+    "pages/static/tools": ".",
   });
 
   config.addFilter("sortByName", (values) => {
     if (!values || !Array.isArray(values)) {
       console.error(`Unexpected values in "sortByName" filter: ${values}`);
       return values;
     }
-    return [...values].sort((a, b) => a.data.name.localeCompare(b.data.name))
+    return [...values].sort((a, b) => a.data.name.localeCompare(b.data.name));
   });
 
   config.setBrowserSyncConfig({
     files: [
-      'pages/dist/bundles/*.js',
-      'pages/dist/bundles/*.css',
-      'pages/dist/bundles/*.map',
+      "pages/dist/bundles/*.js",
+      "pages/dist/bundles/*.css",
+      "pages/dist/bundles/*.map",
     ],
     open: isDev,
   });
 
+  config.addTransform("htmlmin", function (content, outputPath) {
+    if (outputPath && outputPath.endsWith(".html") && !isDev) {
+      return require(`html-minifier`).minify(content, {
```

Comment:
As I see you require html-minifier on 2-nd line and put it in `htmlmin` variable, but here you require it again. Is it correct or we can use `htmlmin` variable from the 2-nd line?

Reply:
Thank you for perfect note.  I've considered your remark and now use htmlmin variable.